src/Forms/ElementalAreaField.php

```diff
@@ -216,85 +216,4 @@ public function setSubmittedValue($value, $data = null)
         // Content comes through as a JSON encoded list through a hidden field.
         return $this->setValue(json_decode($value ?? '', true));
     }
-
```

Comment:
All the code delete here was only use when doing a page/parent form submit.

Inline submits go via ElementController.php instead